test/fixtures/util.go

```diff
@@ -285,3 +355,83 @@ func podLogContains(ctx context.Context, client kubernetes.Interface, namespace,
 		}
 	}
 }
+
+func podLogContainsCount(ctx context.Context, client kubernetes.Interface, namespace, podName, containerName, regex string, count int) (bool, error) {
+	stream, err := client.CoreV1().Pods(namespace).GetLogs(podName, &corev1.PodLogOptions{Follow: true, Container: containerName}).Stream(ctx)
+	if err != nil {
+		return false, err
+	}
+	defer func() { _ = stream.Close() }()
+
+	exp, err := regexp.Compile(regex)
+	if err != nil {
+		return false, err
+	}
+
+	instancesChan := make(chan struct{})
+
+	// scan the log looking for matches
+	go func(ctx context.Context, instancesChan chan<- struct{}) {
+		s := bufio.NewScanner(stream)
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			default:
+				if !s.Scan() {
+					return
+				}
+				data := s.Bytes()
+				fmt.Println(string(data))
```

Comment:
do we need this?

Reply:
This is to print out the watched log, useful for debugging.